.github/workflows/pre-main.yaml

```diff
@@ -68,10 +76,14 @@ jobs:
       - name: make vet
         run: make vet
   shell-linters:
-    name: Shellcheck and shfmt
+    name: Hadolint, shellcheck and shfmt
     runs-on: ubuntu-22.04
     steps:
       - uses: actions/checkout@v3
+      - uses: hadolint/hadolint-action@v3.1.0
```

Comment:
Is it possible to just use this as `v3`?  So we only have to worry about major versions?  See the other actions for example.

Reply:
I've tried without success, it needs the full version number.